src/views/DashboardView.tsx

```diff
@@ -1,23 +1,155 @@
+        pointBackgroundColor: '#fff',
+        pointBorderWidth: 1,
+        pointHoverRadius: 5,
+        pointHoverBackgroundColor: 'rgba(75,192,192,1)',
+        pointHoverBorderColor: 'rgba(220,220,220,1)',
+        pointHoverBorderWidth: 2,
+        pointRadius: 1,
+        pointHitRadius: 10,
+        data: [
+          { x: 10, y: 20, r: 5 },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+          { x: randomNumber(), y: randomNumber(), r: randomNumber() },
+        ],
+      },
+    ],
+  };
+
+  const lineData = {
+    labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+    datasets: [
+      {
+        label: 'My First dataset',
+        fill: false,
+        lineTension: 0.1,
+        backgroundColor: 'rgba(75,192,192,0.4)',
+        borderColor: 'rgba(75,192,192,1)',
+        borderCapStyle: 'butt',
+        borderDash: [],
+        borderDashOffset: 0.0,
+        borderJoinStyle: 'miter',
+        pointBorderColor: 'rgba(75,192,192,1)',
+        pointBackgroundColor: '#fff',
+        pointBorderWidth: 1,
+        pointHoverRadius: 5,
+        pointHoverBackgroundColor: 'rgba(75,192,192,1)',
+        pointHoverBorderColor: 'rgba(220,220,220,1)',
+        pointHoverBorderWidth: 2,
+        pointRadius: 1,
+        pointHitRadius: 10,
+        data: [65, 59, 80, 81, 56, 55, 40],
+      },
+    ],
+  };
+  const Item = ({ Component, data }) => {
+    return (
+      <Card width="400px">
+        <Text h6 style={{ marginBottom: '0' }}>
+          Burundi's main demographic and socio-economic indicators
+        </Text>
+        <Component data={data} />
+        <Card.Footer>
+          <Link block target="_blank" href="#">
+            Download data in CSV
+          </Link>
+        </Card.Footer>
+      </Card>
+    );
+  };
+
   return (
     <Layout pageTitle="Dashboard">
-      <Text h1 style={{ textAlign: 'center', marginTop: '30px' }}>
-        We're Sorry!
+      <Text h4 style={{ textAlign: 'center' }}>
+        POPULATION
       </Text>
-      <Note
-        type="success"
-        style={{
-          width: 'auto',
-          maxWidth: '400px',
-          margin: '0  auto 50px auto',
-        }}
-      >
-        This page is down for maintenance. We are working to get it back up and
-        running as soon as possible. Please check back!
-      </Note>
+      <Grid.Container gap={5} justify="center">
```

Comment:
We have also a scroll on this page I don't understand why is there, maybe is a problem with the Geist UI Grid if you understand why you can try to fix it in a proper way. I have a workaround though.

wrap the GridContainer into a div and give it these styles
```css
 display: 'flex',
 justifyContent: 'center',
 overflowY: 'hidden',
 margin: '50px 0'
```
and give the Grid Container a 100% width so that the scroll on the y-axis can be removed.

Reply:
Thanks for your feedback. It looks nicer now. I'm definitely now good with css.. 🤯